contracts/registry/IZNSRegistry.sol

```diff
@@ -152,12 +152,4 @@ interface IZNSRegistry {
      * @param domainHash The hash of the domain name
      */
     function deleteRecord(bytes32 domainHash) external;
-
-    /**
-     * @notice Set the access controller contract
-     * @param accessController The new access controller
-     */
-    function setAccessController(address accessController) external;
```

Comment:
dangerous thing you are doing. please, let me know why

Reply:
@Whytecrowe what makes this dangerous? We are inheriting it twice, once from the `AccessControlled` contract and then again in the interface for this contract. I was removing the second reference as it seems unnecessary to do this twice, Why do we need this? It's already `virtual` in the `AccessControlled` contract we inherit from so there isn't a possibility of not overriding it as it will error if we don't

Reply:
either remove everywhere or leave this one as well